addons/website_sale/static/src/js/website_sale_tour_shop_cart_recovery.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('website_sale.tour_shop_cart_recovery', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var localStorage = require('web.local_storage');
+    var rpc = require('web.rpc');
+    var tour = require('web_tour.tour');
+
+    var productName = "Acoustic Bloc Screens";
+    var orderIdKey = 'website_sale.tour_shop_cart_recovery.orderId';
+    var recoveryLinkKey = 'website_sale.tour_shop_cart_recovery.recoveryLink';
+
+    tour.register('shop_cart_recovery', {
+        test: true,
+        url: '/shop?search=' + productName,
+        wait_for: base.ready(),
+    },
+    [
```

Comment:
Also check the convention used by other tours for indentation of steps

Reply:
Do we enforce this one? All the other tours I have written in the past months use the format I used here.

Reply:
Well, when there are no clear conventions, I use the one which is used by most other cases if it is not too stupid... so here I would use it. But no, you are not obliged to.